Makefile.rules

```diff
@@ -1268,6 +1271,27 @@ testlibgap: ${LIBGAPTESTS}
 		$(v) -A -l $(top_srcdir) -q -T --nointeract >$(v).out && \
 		diff $(top_srcdir)/$(v).expect $(v).out && ) :
 
+# test libgap's pkg-config's libgap version reporting
+tst/testpkgconfigversion: install-libgap
```

Comment:
This target does not actually create `tst/testpkgconfigversion` and as such should be added to `.PHONY` (and then perhaps renamed `testpkgconfigversion`). 
```suggestion
.PHONY: testpkgconfigversion
testpkgconfigversion: install-libgap
```

Though it also does not set an exit code, so it'll never fail... Perhaps add an `exit 1` in a suitable place?

Or just remove it here and add the same code into `ci.sh`, where it can be rewritten without mangling shell code ;-). (The expected version could then be computed via `make print-GAP_VERSION`)

Reply:
I'd rather keep the make version, as it allows testing locally in a normal way.

Reply:
I don't quite get the removal of tst/ - I thought it's a convention for test targets, no?

Reply:
No other test target has a `tst/` prefix.

And you can invoke `dev/ci.sh` locally, I regularly do it. But as I said, i won't quibble over this

Reply:
> No other test target has a `tst/` prefix.

how about `Makefile.rules:tst/testlibgap/%: ` ? Or `Makefile.rules:tst/testkernel/%:` ?

Reply:
Those are not test targets (in particular, they are not PHONY) but rather regular file targets that are then used by the test targets `testlibgap` and `testkernel`